Formula/scummvm.rb

```diff
@@ -45,8 +45,10 @@ def install
   end
 
   test do
-    # Test fails on headless CI: Could not initialize SDL: No available video device
-    return if OS.linux? && ENV["HOMEBREW_GITHUB_ACTIONS"]
+    if OS.linux?
+      ENV["SDL_VIDEODRIVER"] = "dummy"
+      ENV["SDL_AUDIODRIVER"] = "dummy"
+    end
```

Comment:
Do we need to set this conditionally or will this work on macOS too?

Reply:
I don't see why using a dummy driver on macOS wouldn't work, but I guess there's only one way to find out.